index.html

```diff
@@ -0,0 +1,76 @@
+    <meta name="description" content="Alijan Rahimi's portfolio" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <nav class="navbar">
+                <a class="nav-logo" href="">Rahimi</a>
+                <ul class="nav-menu">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">contact</a></li>
+                </ul>
+                <div class="humburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>       
+        </header>
+        <main class="main">                       
+            <section class="section1">
+                <h1 class="greeting-text">Hey there. I’m <br> Alijan.</h1>
+                <h2 class="profission-text">I’m a software developer</h2>
+                <p class="supporting-text">
+                    I can help you build a product,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                
+                <div class="social-media">
+                    <ul>
+                        <li>
+                            <a href="https://github.com/rahimialijan">
+                                <img src="images/Icons/Icon-GitHub.svg" alt="GitHub">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/alijan-rahimi-18389ab3/">
+                                <img src="images/Icons/Icon-Linkedin.svg" alt="Linkedin">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                <img src="images/Icons/Icon-Angelist.svg" alt="Angelist">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://twitter.com/AlijanRahimi10">
+                                <img src="images/Icons/Icon-Twitter.svg" alt="Twitter">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="">
+                                <img src="images/Icons/Icon-Medium.svg" alt="Medium">
+                            </a> 
+                        </li>
+                    </ul>
+                </div>
+                <div class="action-arrow-down">
+                    <a href="#recent-work">
+                        <img src="images/Icons/Icon-Arrow-button.svg" alt="action button arrow down">
+                    </a>
```

Comment:
- [ ] [OPTIONAL] You have done a great job to have an arrow button. However, since we have to stick to the Figma design as much as possible, let's make sure that the arrow does not overlap with the other contents of your headline section ☺️ 

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/227128757-15633628-9877-4189-bef6-a5e67c322afa.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/227128872-1860b5e3-ac2b-4da1-999b-9d03c35338ee.png)